tikv-client/src/main/java/com/pingcap/tikv/types/Converter.java

```diff
@@ -176,4 +180,60 @@ public static BigDecimal convertToBigDecimal(Object val) {
       throw new TypeException("can not cast non Number type to Double");
     }
   }
+
+  public static String convertDurationToStr(long nanos, int decimal) {
```

Comment:
These classes need separated tests and please make sure have 100% line coverage.

Reply:
DONE